packages/storybook/src/components/designTokens/DesignTokens.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { TokensTable } from '../tokensTable/TokensTable';
+import { TokenPreview } from '../tokenPreview/TokenPreview';
+import { Heading } from '../heading/Heading';
+import { extractTokens } from '../../utils/tokenUtils';
+import './DesignTokens.scss';
+
+const categoryDescriptions: { [key: string]: string } = {
+  'Colors': 'Colors in our design system are organized into several categories including palettes, semantic colors, brand colors, and states.',
+  'Font Family': 'Font families define the typefaces used throughout the application, including default text and monospace code.',
+  'Border Radius': 'Border radius tokens help maintain consistent corner rounding across components.',
+  'Border Width': 'Border width tokens define consistent border thicknesses.',
+  'Breakpoints': 'Breakpoint tokens define consistent responsive breakpoints for building responsive layouts.',
+  'Outline': 'Outline tokens define the appearance of focus indicators for accessibility. Each token contributes to the complete outline style.',
+  'Transition': 'Transition tokens ensure consistent animation timing across components.',
+};
+
+export const DesignTokens: React.FC = () => {
+  const categories = extractTokens();
```

Comment:
should be done once in a `useEffect`, and all the logic inside:
- `category.name.toLowerCase().replace(/\s+/g, '-');`
- token transformation